scripts/us_ca_chhs/healthcare_facility_bed_counts/preprocess_csv.py

```diff
@@ -0,0 +1,261 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pandas as pd
+import csv
+import os
+from preprocess_csv_helper import preserve_leading_zero
+from preprocess_csv_helper import generate_dcid_for_county
+
+
+# process dataset 1
+df1 = pd.read_excel('https://data.chhs.ca.gov/dataset/09b8ad0e-aca6-4147-b78d-bdaad872f30b/resource/0997fa8e-ef7c-43f2-8b9a-94672935fa60/download/healthcare_facility_beds.xlsx',
+                    converters={'FACID': lambda x: str(x)})
+
+new_columns = ['GeoId', 'Date',
```

Comment:
We generally prefer having all the constants defined at the top, before doing any actual calls like pd.read_excel.